src/test/regression/3100/regression-3100-b.yaml

```diff
@@ -0,0 +1,62 @@
+general:
+  stop_time: 60 seconds
```

Comment:
Maybe explicitly disable `model-unblocked-syscall-latency` in case the default switches in the future, since having it enabled can mask this issue by inserting a delay between the send and the close.

Reply:
Ah good point. I'll add this to both regression tests (3100-a and 3100-b).